README.md

```diff
@@ -17,50 +23,26 @@ curl https://raw.githubusercontent.com/rcaloras/bash-preexec/master/bash-preexec
 ```bash
 git clone https://github.com/otto-aa/aw-watcher-bash
 cd aw-watcher-bash
-make build
+make install
 ```
 
+### Run aw-watcher-terminal in the background
+
+Make sure that aw-watcher-terminal is running in the background.
+This is needed because aw-watcher-bash is just a hook which sends terminal info to aw-watcher-terminal, the data itself is then packaged and sent by aw-watcher-terminal so without that running all data will be lost.
+
 ### Add following code to the bottom of your ~/.bashrc file
 
-_Note: In case you use zsh, drop the bash-preexec.sh part, as preexec is in-build in zsh_
+Replace the PATH variable to the path to the aw-watcher-bash-preexec.sh script on your computer
```

Comment:
Writing `PATH` variable could lead to confusin with the in-built PATH.

I'd suggest to just write it out:
Replace $PATH_TO_AW_WATCHER_BASH_PREEXEC.SH with the path to the aw-watcher-bash-preexec.sh script on your computer.

Reply:
True, fixed